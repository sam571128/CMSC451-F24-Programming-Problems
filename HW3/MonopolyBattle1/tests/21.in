37 61 146
6 35
24 31
6 33
6 17
12 18
3 32
23 30
32 3
35 6
3 19
1 12
9 26
26 11
35 18
4 4
3 32
19 14
35 31
6 36
11 20
6 7
37 36
16 2
30 31
8 1
8 36
33 13
16 23
27 11
29 32
17 30
12 22
8 2
14 4
34 6
7 30
25 15
20 12
15 36
29 8
11 37
8 15
18 27
17 13
11 25
11 34
15 29
30 1
24 20
7 23
31 34
29 35
37 28
25 17
19 6
37 2
28 9
30 2
10 3
30 23
31 31
