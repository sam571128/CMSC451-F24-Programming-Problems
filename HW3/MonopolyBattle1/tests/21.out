47 47 47 47 -1 46 46 47 57 -1 46 43 46 47 45 -1 46 46 47 46 -1 46 50 -1 46 57 46 53 47 46 46 47 46 46 46 43 46 